src/WebJobs.Script.Grpc/Channel/GrpcCapabilities.cs

```diff
@@ -26,18 +26,34 @@ public string GetCapabilityState(string capability)
             return null;
         }
 
-        public void UpdateCapabilities(IDictionary<string, string> capabilities)
+        public void UpdateCapabilities(IDictionary<string, string> capabilities, GrpcCapabilitiesUpdateStrategy strategy)
         {
             if (capabilities == null)
             {
                 return;
             }
 
-            _logger.LogDebug($"Updating capabilities: {capabilities.ToString()}");
+            if (strategy == GrpcCapabilitiesUpdateStrategy.Merge)
```

Comment:
Can this be a switch statement instead? I think it could look a little nicer and makes it easier to add new statements if we add new strategies.